Generals/Code/GameEngine/Source/GameClient/GUI/LoadScreen.cpp

```diff
@@ -492,6 +506,15 @@ void SinglePlayerLoadScreen::init( GameInfo *game )
 		{
 			TheGameEngine->serviceWindowsOS();
 
+			if( TheKeyboard )
+			{
+				TheKeyboard->UPDATE();
+				TheKeyboard->createStreamMessages();
+			}
+
+			if( TheMessageStream )
+				TheMessageStream->propagateMessages();
```

Comment:
Format: Maybe there can be more consistency in the way this is written. The other place has it written like so:

```cpp
			if ( TheKeyboard )
			{
				TheKeyboard->UPDATE();
				TheKeyboard->createStreamMessages();
			}
			if ( TheMessageStream )
			{
				TheMessageStream->propagateMessages();
			}
```

Consistent formatting makes it easier to eyeball.